src/items/rqgItem.ts

```diff
@@ -160,4 +167,10 @@ export class RqgItem extends Item {
     const okToAdd = !isRuneMagic || !(isRuneMagic && !actorHasCult);
     return !okToAdd;
   }
+
+  //** Localizes Item Type Name using ITEM localization used by Foundry.
+  //** See also  */
```

Comment:
see also what?

Reply:
Removed.  I was trying to cross-reference to the RqgItem to indicate that they both did a similar localization but then I had the better idea of just using the RqgItem localization in the handlebar helper.